bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/binding/generic/ChannelTransformation.java

```diff
@@ -71,15 +71,47 @@ private static Stream<String> splitTransformationString(String transformationStr
         return Arrays.stream(transformationString.split("∩")).filter(s -> !s.isBlank());
     }
 
+    /**
+     * Checks whether the given string contains valid transformations.
+     * 
+     * Valid single and chained transformations will return true.
+     * 
+     * @param value the transformation string to check.
+     * @return <code>true</code> if the string contains valid transformations, <code>false</code> otherwise.
+     */
+    public static boolean isTransform(@Nullable String value) {
```

Comment:
Please move this down and place it before the `TransformationStep` class. I think `isValidTransformation` would be a better naming.

Reply:
I was following the naming from the other classes. Renamed as suggested.